### PR TITLE
Change shiftMF from a WarpX public static function to a public member function of the WarpX class

### PR DESCRIPTION
**Pull request by lucafedeli88**
This PR is part of the effort to simplify the WarpX class.

### CODE REVIEW

File: Source/WarpX.H [hunk 1]
```diff
@@ -1137,6 +1132,11 @@ public:
 
     FiniteDifferenceSolver * get_pointer_fdtd_solver_fp (int lev) { return m_fdtd_solver_fp[lev].get(); }
 
+    void shiftMF (amrex::MultiFab& mf, const amrex::Geometry& geometry,
```
Comment (ax3l) [inline Source/WarpX.H:1135]: Let's add a Doxygen string?

**Review by ax3l — COMMENTED** [review]
I do not understand how it simplifies the class yet o.0

It is not static anymore, but still part of the public class interface and still using static member variables.

**Comment by ax3l** [comment]
ping @lucafedeli88 :)

**Comment by lucafedeli88** [comment]
I will close this PR for the moment, since it is not a particularly useful refactoring of the code.

**Pull request closed without merging**